library/alloc/src/rc.rs

```diff
@@ -1153,7 +1153,7 @@ impl<T: ?Sized> Rc<T> {
     #[inline]
     #[stable(feature = "ptr_eq", since = "1.17.0")]
     /// Returns `true` if the two `Rc`s point to the same allocation in a vein similar to
-    /// [`ptr::eq`]. See [that function][`ptr::eq`] for caveats when comparing `dyn Trait` pointers.
+    /// [`ptr::eq`]. However, this function ignores the metadata of  `dyn Trait` pointers.
```

Comment:
Slight language nit: the "However" should be removed. Simply saying "This function ignores the metadata of ..." is enough.